.github/workflows/build.yaml

```diff
@@ -25,18 +25,18 @@ jobs:
           cp SourceHanSansTW/SubsetOTF/TW/*.otf /usr/share/fonts/
           fc-cache -fv
       - name: Setup Typst
-        uses: yusancky/setup-typst@v2
+        uses: typst-community/setup-typst@v4
         id: setup-typst
         with:
-          version: 'latest'
+          typst-version: 'latest'
```

Comment:
Hi there. You may need to set `typst-version` here to specify the version to use.